common/errors/errors.go

```diff
@@ -14,3 +21,38 @@ type ApiError struct {
 	Message  string `json:"message"`
 	RawError string `json:"raw_error,omitempty"`
 }
+
+type ErrorLogEntry struct {
+	ID    string
+	Error interface{}
+}
+
+func LogError(id string, error_message interface{}) {
+	log_entry := ErrorLogEntry{
+		ID:    id,
+		Error: error_message,
+	}
+
+	error_log_message, err := json.MarshalIndent(log_entry, "", "    ")
```

Comment:
Hard to tell what this code is doing without looking up what `MarshalIndent` means; maybe put the "tab" string in a constant called `INDENT`?